inst/templates/mapping_template_CAMPAIGNers.csv

```diff
@@ -811,22 +811,22 @@ NA;2;carbon management;Carbon Capture|Storage|Electricity;Mt CO2/yr;Carbon Manag
 770;2;energy (final);Final Energy|Industry|Non-Ferrous Metals|Geothermal;EJ/yr;;;;;;Final energy consumption of geothermal heat by the non-ferrous metals sector;
 771;2;energy (final);Final Energy|Industry|Non-Ferrous Metals|Waste;EJ/yr;;;;;;Final energy consumption of non-renewable waste by the non-ferrous metals sector;
 772;2;energy (final);Final Energy|Industry|Non-Ferrous Metals|Other;EJ/yr;;;;;;Final energy consumption of other energy carriers by the non-ferrous metals sector (please provide a definition of the energy carriers in this category in the 'comments' tab);
-773;1;energy (final);Final Energy|Residential;EJ/yr;;;;not modelled but post-processed;;final energy consumed in the residential sector;
-774;2;energy (final);Final Energy|Residential|Electricity;EJ/yr;;;;not modelled but post-processed;;final energy consumption by the residential sector of electricity (including on-site solar PV), excluding transmission/distribution losses;
```

Comment:
same…

Reply:
That was a mistake, thank you for pointing this out!